src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -2153,3 +2153,40 @@ def test_fw_operand_clause(fortran_writer):
     ref_a = Reference(symbol_a)
     op_clause.addchild(ref_a)
     assert "depend(inout: a)" in fortran_writer(op_clause)
+
```

Comment:
I know it's only one test but rather than keep adding to this large file, please could you break it out.

Reply:
Done